### PR TITLE
feat: update requirement content for Native and Custom Widgets, Instructions requirement

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

Update Instructions requirement content for Native Widgets and Custom Widgets.

**ORIGINAL: native widgets -> instructions**
01 - original native widgets - instructions - content

**UPDATED: native widgets -> instructions**
03 - updated native widgets - instructions - content

**ORIGINAL: custom widgets -> instructions**
02 - original custom widgets - instructions - content

**UPDATED: custom widgets -> instructions**
04 - updated custom widgets- instructions - content

#### Pull request checklist

- [x] Addresses an existing issue: WI # 1587568 and # 1587567
- [x] Added relevant unit test for your changes. (`yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/assessments/common/instructions-and-labels-note.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+import * as Markup from '../markup';
+
+export const InstructionsAndLabelsNotes = () => (
+    <Markup.Emphasis>
+        Note: For WCAG 2.1 compliance both visible instructions and labels must be programatically related to a widget. For WCAG 2.0
```
Comment (lisli1) [inline src/assessments/common/instructions-and-labels-note.tsx]: added comma after "WCAG 2.1 compliance" and updated spelling of "programmatically"
```suggestion
        Note: For WCAG 2.1 compliance, both visible instructions and labels must be programmatically related to a widget. For WCAG 2.0
```

File: src/assessments/common/instructions-and-labels-note.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        compliance, only visible instructions must be programatically related to a widget.
```
Comment (lisli1) [inline src/assessments/common/instructions-and-labels-note.tsx]: Spelling here too. I'm getting red underline for this word in the comment editor but according to [Merriam-Webster dictionary](https://www.merriam-webster.com/dictionary/programmatic#other-words), this is how it's spelled.
```suggestion
        compliance, only visible instructions must be programmatically related to a widget.
```

**Comment by lisli1** [comment]
It seems like there are some minor UI changes. Should we include screenshots? I can also see an argument for not including them (they are just text changes).

File: src/assessments/native-widgets/test-steps/instructions.tsx [hunk 3]
```diff
@@ -1,44 +1,49 @@
...
+);
 
 const howToTest: JSX.Element = (
     <div>
         <p>
-            For this requirement, {productName} highlights native widgets. Native widgets include
+            The visual helper for this requirement highlights native widgets. Native widgets include
             <Markup.NonBreakingSpace />
-            <Markup.Tag tagName="button" isBold={true} />,
+            <Markup.Tag tagName="button" isBold={false} />,
```
Comment (lisli1) [inline src/assessments/native-widgets/test-steps/instructions.tsx]: Just wondering, does Markup.Tag default to bold?

Comment (smoralesd) [reply src/assessments/native-widgets/test-steps/instructions.tsx]: yes... I remove the `isBold` first, and found out the tags we're still bold. I check the code and yes, we are explicitly defaulting to boldness on `Markup.Tag`

**Pull request merged**